extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -212,31 +214,37 @@ class MembershipsButtonEdit extends Component {
 			return;
 		}
 
+		const minPrice = formatCurrency(
+			minimumTransactionAmountForCurrency( this.state.editedProductCurrency ),
+			this.state.editedProductCurrency
+		);
+		const minimumPriceNote = `${ __( 'Minimum allowed price is ', 'jetpack' ) } ${ minPrice }`;
```

Comment:
Do you think you could rely on `sprintf` here and include the price in the translatable string? I think that would make things easier for translators.

Reply:
Yes, was looking for the recommendation. [The handbook was a bit ambiguous as to handle interpolation](https://developer.wordpress.org/block-editor/developers/internationalization/).